src/app/multi-editor/multi-editor.component.ts

```diff
@@ -264,6 +251,40 @@ export class MultiEditorComponent implements OnInit {
     this.changeDetectorRef.markForCheck();
   }
 
+  private checkExistenceOfAction(userActions: UserActions): boolean {
+    let existingAction = false;
+    return userActions.actions.some((element, index, array) => {
+      if (element.mainKey) {
+        return true;
+      }
+    });
+  }
+
+  private showError(error: any) {
+    const errorMessage = error.json().message;
+    if (errorMessage) {
+      this.toastr.error(errorMessage);
+    } else {
+      this.toastr.error(error);
+    }
+  }
+
+  private validateActionsKeypaths(userActions: UserActions): boolean {
+    let invalid = false;
+    userActions.actions.some((element, index, array) => {
+      if (!this.schemaKeysStoreService.findSubschema(element.mainKey)) {
+        return true;
+      }
+    });
+    if (!invalid) {
+      userActions.conditions.forEach((element, index, array) => {
+        if (!this.schemaKeysStoreService.findSubschema(element.key)) {
```

Comment:
`.some(condition => !this.schemaKeysStoreService.findSubschema(element.key))` is better. Because returning boolean in an `if` block is always unnecessary since you can always return the `if` condition itself